hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ClusterStateEndpoint.java

```diff
@@ -81,20 +82,26 @@ public Response getClusterState() {
             stats.getScmUsed().get(), stats.getRemaining().get());
     ClusterStateResponse.Builder builder = ClusterStateResponse.newBuilder();
     try {
-      builder.setVolumes(
-          omMetadataManager.getVolumeTable().getEstimatedKeyCount());
+      Table volumeTable = omMetadataManager.getVolumeTable();
```

Comment:
Instead of not null checks, can we have a state in MetadataManager like INITIALIZED, STOPPED? This will help be future proof, and if anywhere this check is required it will be generic.

Reply:
Thanks for the review! I added a boolean flag to keep track of the initialized state to ReconOMMetadataManager.